### PR TITLE
Change alt text for Business Image on Credit Page

### LINKED ISSUE #1561: Change Alt Text for Business Image on Credit page
### Overview
Alt text for Business Image on Credit Page needs update.

### Action Items
- [ ] Change the alt attribute value to 'One-on-one video conference concept illustration'

```diff
- alt: 'Image of Business'
+ alt: 'One-on-one video conference concept illustration'
```

### Resources/Instructions
https://github.com/hackforla/website/blob/90a20d1e088b0a05a286af101f82135ad4774e39/_data/internal/credits/business.yml#L10

### PR DESCRIPTION
**Pull request by blakes24**
Fixes #1561

### What changes did you make and why did you make them ?

  -  Changes the alt attribute value in business.yml to 'One-on-one video conference concept illustration'

### Screenshots of Proposed Changes Of The Website (if any, please do not screen shot code changes)

<details>
<summary>Alt text before</summary>
<br>
Screen Shot 2021-07-26 at 11 22 41 AM
</details>
<details>
<summary>Alt text after</summary>
<br>
Screen Shot 2021-07-26 at 11 19 50 AM
</details>

### CODE REVIEW

**Review by adam-abundis — CHANGES_REQUESTED** [review]
@blakes24 Good job on your first pull request. I would remove the changes made within `pages/credits.html`. Remember that you only make changes within the `Action Items` of your issue. Those changes would resolve #1970, but we try to match 1 pull request to 1 issue.

@Aveline-art @ExperimentsInHonesty How should we proceed with this pull request?

File: pages/credits.html [hunk 1]
```diff
@@ -24,7 +24,7 @@ <h1>Credits</h1>
                           <!-- originally class="project-card-inner" -->
                         <div class="credits-item">
                           <div class="icon-left">
-                            <img src={{ item[1].image-url | absolute_url }} alt={{item[1].alt}}/>
+                            <img src={{ item[1].image-url | absolute_url }} alt="{{item[1].alt}}"/>
```
Comment (adam-abundis) [inline pages/credits.html]: After talking with @Aveline-art, let's remove this change. The guideline is 1 pull request for 1 issue.

Comment (blakes24) [reply pages/credits.html]: Ok, I have removed the change.

**Review by JackRichman — APPROVED** [review]
This is great! I ran it on docker from my local repository and it looks good.

**Pull request merged**